spring-rabbit/src/main/java/org/springframework/amqp/rabbit/AsyncRabbitTemplate.java

```diff
@@ -399,6 +403,57 @@ public RabbitMessageFuture sendAndReceive(String exchange, String routingKey, Me
 	@Override
 	public <C> RabbitConverterFuture<C> convertSendAndReceive(String exchange, String routingKey, Object object,
 			MessagePostProcessor messagePostProcessor) {
+		return convertSendAndReceive(exchange, routingKey, object, messagePostProcessor, true);
+	}
+
+	@Override
+	public <C> RabbitConverterFuture<C> convertSendAndReceiveAsType(Object object,
+			ParameterizedTypeReference<C> responseType) {
+		return convertSendAndReceiveAsType(this.template.getExchange(), this.template.getRoutingKey(), object,
+				(MessagePostProcessor) null, responseType);
```

Comment:
Well - perhaps when they fix "unused imports" and "unused variables" like that one you left in yesterday 😄